### DISCUSSION TITLE
Question about hiding columns

### DISCUSSION THREAD

**Discussion by isak102** [davatorium/rofi#1646] [General]
In [this issue](https://github.com/davatorium/rofi/issues/680) someone asks about hiding columns similar to the option `--with-nth` in fzf, and then the dev replies [here](https://github.com/davatorium/rofi/issues/680#issuecomment-1025663381) that it is already implemented. I tried looking for it in the documentation but I couldn't find anything, can someone help me here? This option would be really useful. Thanks

**Comment by DaveDavenport** [comment]
Yes, see manpage (as mentioned in the link): https://github.com/davatorium/rofi/blob/next/doc/rofi-dmenu.5.markdown under `-display-columns`.

**Reply by isak102** [reply]
Ah, I missed that. Thank you very much
